app/controllers/profile/posts_controller.rb

```diff
@@ -8,4 +8,19 @@ def show
     @post = current_user.posts.find(params[:id])
   end
 
+  def create
+    day = Day.create(week: 1, day_of_week: 1, group: current_user.group)
```

Comment:
```suggestion
    day = Day.find_or_create_by(week: 1, day_of_week: 1, group: current_user.group)
```
We don't want to create copies of days